en/donate.html

```diff
@@ -44,7 +44,38 @@
         </div>
     </nav>
     
-    <main class="bg-gray-100"></main>
+    <main class="bg-gray-100">
+
+        <div class="grid items-center md:grid-cols-[repeat(2,_1fr)] w-full relative mb-0">
+            <img class="shadow-2xl rounded-tr-md" src="../assets/katt-yukawa-K0E6E0a0R3A-unsplash 1.png" alt="An assortment of vegetables">
+            <div class="flex flex-col items-center text-center">
+                <p class="mb-5 text-2xl font-normal text-black uppercase font-bold w-[70%]">WELLFOOD</p>
+                <p class="text-2xl font-normal text-midnight uppercase font-bold w-[70%]">Make a donation</p>
+                <div class="mt-10">
+                    <button type="button" class=" text-gray-900 hover:text-white border border-gray-800 hover:bg-gray-900 focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center mr-2 mb-2 dark:border-gray-600 dark:text-gray-400 dark:hover:text-white dark:hover:bg-gray-600 dark:focus:ring-gray-800">$20</button>
```

Comment:
Removing a few of the `dark-` classes since we don't have a dark mode and it sort of clutters up the utility classes, not a big deal though (I won't make the suggestion for all, might do a global search/remove later)
```suggestion
                    <button type="button" class="text-gray-900 hover:text-white border border-gray-800 hover:bg-gray-900 focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center mr-2 mb-2">$20</button>
```